modules/react/icon/lib/AccentIcon.tsx

```diff
@@ -43,23 +47,46 @@ export const accentIconStyles = ({
   },
 });
 
+const accentIconStencil = createStencil({
+  vars: {
+    color: `${base.blueberry500}`,
+  },
+  base: ({color}) => ({
+    '& .color-500': {
+      fill: color,
+    },
+    '& .french-vanilla-100': {
+      fill: base.frenchVanilla100,
+    },
+  }),
+  modifiers: {
+    transparent: {
+      true: {
+        '& .french-vanilla-100': {
+          fill: 'transparent',
+        },
+      },
+    },
+  },
+});
+
 export const AccentIcon = createComponent('span')({
   displayName: 'AccentIcon',
   Component: (
-    {transparent = false, size = 56, icon, color, shouldMirror, ...elemProps}: AccentIconProps,
+    {transparent = false, size = 56, icon, color, ...elemProps}: AccentIconProps,
```

Comment:
I've been wondering about defaults. There's `defaultModifiers` that builds the defaults directly into the stencil. This would also be helpful for documentation of the stencil to communicate what's the expected default of the stencil and not just the React component. You'd add the following and remove the defaults here:

```tsx
const accentIconStencil = createStencil({
  vars: {
    color: `${base.blueberry500}`,
  },
  base: ({color}) => ({
    [svgStencil.vars.size]: px2rem(56), // default size built into base
    '& .color-500': {
      fill: color,
    },
    '& .french-vanilla-100': {
      fill: base.frenchVanilla100,
    },
  }),
  modifiers: {
    transparent: {
      true: {
        '& .french-vanilla-100': {
          fill: 'transparent',
        },
      },
    },
  },
  // don't need a default for `transparent`, there's no `undefined` modifier, so it will never match
});
```

Then you can get rid of the defaults:
```suggestion
    {transparent, size, icon, color, ...elemProps}: AccentIconProps,
```

Reply:
The benefit of defaulting a CSS variable in the `base` style is the component doesn't need a `style` attribute for every single `AcentIcon` component. It will be the default automatically.